generic_expression_patterns_modules/process.py

```diff
@@ -449,12 +429,12 @@ def process_raw_compendium_pseudomonas(
     # Create processed pseudomonas compendium data file
     raw_compendium = pd.read_csv(raw_filename, header=0, index_col=0, sep="\t")
 
-    if raw_compendium.shape != (950, 5549):
+    if raw_compendium.shape[1] != 5549:
```

Comment:
Is there a reason that there will always be exactly this many genes/samples?

Reply:
This is the number of genes specific to this pseudomonas compendium, which this function is designed to process data for. I think I will add a comment to explain that though so a user doesn't mistake this for any pseudomonas dataset.